src/components/Main.jsx

```diff
@@ -0,0 +1,40 @@
+import { useState, useEffect } from "react";
+import { getAllArticles } from "../../api";
+
+function Main() {
+  const [articles, setArticles] = useState([]);
+
+  useEffect(() => {
+    getAllArticles().then((data) => {
+      setArticles(data);
+    });
+  }, []);
+
+  return (
+		<section>
+			<h1 className="pt-10 text-3xl">Available articles</h1>
+			<ol>
+			{articles.map((article) => {
+				return (
+					<li className="pt-10" key={article.article_id}>
+					<h2 className="font-bold inline-break break-all w-100">{article.title}</h2>
+						<div className="flex">
+						<img src={article.article_img_url} alt="article image" width='300px' />
+						<div className="flex flex-col gap-5 pl-10 pt-5">
+							<p>Topic: {article.topic}</p>
+							<p>Votes: {article.votes}</p>
+							<p>{article.created_at.slice(0,19).replace("T", " ")}</p>
+							<p>Comments: {article.comment_count}</p>
+						</div>
+						</div>
```

Comment:
Great that you've shown a good amount of info here, with some decent styling
Great that you're presenting a lot of useful info, especially as we'll be asking you to be able to sort the articles by things like notes, comment count etc.
But it may be worth having a think about how you're presenting your information - this article is a very good read when we're considering these things https://www.refactoringui.com/previews/labels-are-a-last-resort
Also, you could consider putting the article cards into their own component, it could help compartmentalise your code a bit more 👀